src/bash/0001-Add-plugin-support-to-bash.patch

```diff
@@ -0,0 +1,222 @@
+From b4962d09ab2d41a5e993bf3d8e4ebe32114999dc Mon Sep 17 00:00:00 2001
+From: liuh-80 <58683130+liuh-80@users.noreply.github.com>
+Date: Fri, 3 Sep 2021 14:05:36 +0800
+Subject: [PATCH] Add plugin support to bash.
```

Comment:
>bash [](http://example.com/codeflow?start=39&length=4)

Please also submit a PR to upstream.

Reply:
After check with liuxin about the license issue, will submit PR to bash.